fdbserver/ConfigDatabaseUnitTests.actor.cpp

```diff
@@ -265,7 +279,7 @@ class BroadcasterToLocalConfigEnvironment {
 
 	void compact() { broadcaster.compact(lastWrittenVersion); }
 
-	Future<Void> getError() const { return readFrom.getError() || broadcastServer; }
+	Future<Void> getError() const { return readFrom.getError(); }
```

Comment:
Why is `broadcastServer` removed here?

Reply:
The Future returned from starting the config broadcaster returns as soon as the initial set of changes have been pushed to the client instead of looping forever like in the old model. So this function will return an error if it continues to listen to `broadcastServer`.

Reply:
With the new implementation, I think there should be a new public `ConfigBroadcaster::getError()` method exposed, and this can be added to the `getError` implementation here.